src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -65,8 +65,36 @@
     <SignedPackageFile Include="$(SdkOutputPath)tools\netcoreapp2.0\Newtonsoft.Json.dll" Certificate="$(AssemblySigning3rdPartyCertName)" />
 
     <!-- Binaries that should be signed by corefx/roslyn -->
-    <ExcludePackageFileFromSigning Include="$(SdkOutputPath)tools\netcoreapp2.0\Microsoft.CodeAnalysis.CSharp.dll" />
     <ExcludePackageFileFromSigning Include="$(SdkOutputPath)tools\netcoreapp2.0\Microsoft.CodeAnalysis.dll" />
+    <ExcludePackageFileFromSigning Include="$(SdkOutputPath)tools\netcoreapp2.0\cs\Microsoft.CodeAnalysis.resources.dll" />
```

Comment:
Same deal. Wildcards don't work here.

Reply:
By design. Wildcards have burned us several times in the past since they are too flexible and pick up things for code signing which should be signed, and vice versa.